internal/communication/mailer/message.go

```diff
@@ -0,0 +1,56 @@
+package mailer
+
+import (
+	"errors"
+	"strconv"
+	"time"
+)
+
+type MessageType uint
```

Comment:
If I not mistaken, you could make `MessageType` of `string` type and then you don't need serialize/deserialize it in json message

Reply:
But what is the purpose send `string` in message?

Reply:
Indeed, this way of storing `MessageType` is much more simpler and obvious. Changed `MessageType` to string.